feat: adds additional configuration options to DRACOLoader

NewYearNewPhil opened the pull request:
three.js [docs](https://threejs.org/docs/#examples/en/loaders/DRACOLoader) suggest reusing  a single DRACOLoader instance, this PR does that

the options exposed in `ThDRACOLoader.ts` still work as expected

demike commented:
@NewYearNewPhil thank you for your contribution.

Would it be more clear to store the instance in the draco loader service?
In addition this service could have flag like
`shareLoader` ( default: true )

What do you think?

NewYearNewPhil commented:
@demike Yes, makes total sense. I've changed the implementation to match.
While looking trough the three.js docs I also noticed there's an option to set a `workerLimit` to DRACOLoader, I've gone ahead and exposed it as well. Renamed the PR to fit the purpose.

demike commented:
Hi looks good. Just one small 'trailing space' linter issue. But then we are good to go!

demike commented:
@NewYearNewPhil Nice! Thank you! 🎉
